nbdime/nbmergeapp.py

```diff
@@ -48,12 +48,12 @@ def main_merge(args):
     else:
         nbdime.log.debug("Merge completed successfully with no unresolvable conflicts.")
 
-    if args.decisions:
+    if decisions:
         # Print merge decisions (including unconflicted)
         out = io.StringIO()
         pretty_print_merge_decisions(b, decisions, out=out)
         nbdime.log.warning("Conflicts:\n%s", out.getvalue())
-    elif mfn:
+    if mfn:
```

Comment:
This was also deliberate, if --decisions are asked for, it doesn't print the merged notebook.